kolibri/plugins/setup_wizard/assets/src/views/ImportIndividualUserForm.vue

```diff
@@ -169,12 +173,22 @@
         return '';
       },
       modalMessage() {
-        return this.$tr('deviceLimitationsMessage', {
+        const importedUserIsAdmin = this.roles && this.roles.includes('admin');
+
+        const messageArgs = {
           full_name: this.full_name,
           username: this.username,
-          roles: this.roles,
+          non_admin_role:
+            !importedUserIsAdmin &&
+            this.coreString('coachLabel').toLocaleLowerCase(currentLanguage),
```

Comment:
Non blocking: It doesn't look like we get to use the `toLocaleLowerCase` that much, its actually a first 😄. Because of its benefits with localization strings, It might be good to implement it as a utility function that will rid developers of the need to  import and pass the currentLanguage every time they need a localized lower case string

Reply:
this is a great point, @akolson! i realized it wasn't present in the codebase (or at least i couldn't find it haha) but sought it out because i was a bit nervous `.toLowerCase`ing a potentially-translated string. i think this is a great suggestion that could be its own little Issue, if this case is encountered elsewhere or we just want to have it in place